webcompat/webhooks/helpers.py

```diff
@@ -47,13 +47,17 @@ def extract_browser_label(metadata_dict):
         return None
 
 
-def extract_extra_label(metadata_dict):
-    """Return the 'extra' label from metadata_dict."""
-    extra_label = metadata_dict.get('extra_label', None)
-    if extra_label:
-        extra_label = extra_label.lower()
-        extra_label = extra_label.encode('utf-8')
-    return extra_label
+def extract_extra_labels(metadata_dict):
+    """Return the 'extra_labels' labels from metadata_dict, as a list."""
+    # we have to handle the ', ' foo biz here.
+    extra_labels = metadata_dict.get('extra_labels', None)
+    if extra_labels:
+        extra_labels = extra_labels.split(', ')
+        print(extra_labels)
```

Comment:
these three lines are a bit dangerous. They are changing the type of the variable.
```python
labels = metadata_dict.get('extra_labels', None)
if labels:
    extra_labels = labels.split(', ')
```

(which also makes me thing if we should return an empty list instead of `None` I wonder if that would simplify things down the pipe. But not urgent)


```

Reply:
I prefer `None`, but I don't think it makes much of a difference...since we need to filter out `None` anyways for `priority_label`. 

If someone feels strongly, please file a follow up bug. :)